index.html

```diff
@@ -3,6 +3,19 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!--iOS-->
+    <link rel="apple-touch-icon" href="/images/iOS/touch-icon-iphone.png" >
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad.png" sizes="76x76">
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-iphone-retina.png" sizes="120x120">
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad-retina.png" sizes="152x152">
+    <meta name="apple-mobile-web-app-title" content="Abir en whatsapp"> 
```

Comment:
typo

Reply:
Ese es el nombre de la apliacion cuando se agrega al inicio! ud prefiere que esta se muestre en ingles?

Reply:
English please

Reply:
I mean you have a typo here, it says "Abir" instead of "Abrir"

Reply:
ok, do you prefer the text in english or spanish?

Reply:
Spanish is ok for now

Reply:
ok